### PR TITLE
[R20-535] Remove duplicate component imports

### PR DESCRIPTION
**Pull request by waitingallday**
**Issue**: https://digital-vic.atlassian.net/browse/R20-535

### What I did

- Create additional vite build for components, update all files to use new endpoint (slight reworking in nuxt, content types, storybook)
- Update all packages to use `workspace:*`

### How to test

- All of the tests
- 

#### For all PR's

- [ ] I've added relevant changes to the project Readme if needed.
- [ ] I've updated the documentation site as needed.
- [ ] I have added unit tests to cover my changes (if not applicable, please state why in a comment)

#### For new components only

- [ ] I have added a story covering all variants
- [ ] I have checked a11y tab in storybook passes
- [ ] Any events are emitted on the event bus

### CODE REVIEW

File: packages/nuxt-ripple/components/TideBaseLayout.vue [hunk 1]
```diff
@@ -62,7 +88,7 @@
 <script setup lang="ts">
 // @ts-ignore
 import { useHead, useSiteTheme, useAppConfig, useRoute } from '#imports'
-import { RplChip } from '@dpc-sdp/ripple-ui-core'
+import { RplChip } from '@dpc-sdp/ripple-ui-core/vue'
```
Comment (dylankelly) [inline packages/nuxt-ripple/components/TideBaseLayout.vue:91]: import from `#components`

File: packages/nuxt-ripple/components/TideBreadcrumbs.vue [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+  <RplBreadcrumbs
+    v-if="breadcrumbs"
+    :items="breadcrumbs"
+    data-cy="breadcrumbs"
+  />
 </template>
 
 <script setup lang="ts">
-import { RplBreadcrumbs } from '@dpc-sdp/ripple-ui-core'
+import { RplBreadcrumbs } from '@dpc-sdp/ripple-ui-core/vue'
```
Comment (dylankelly) [inline packages/nuxt-ripple/components/TideBreadcrumbs.vue:10]: import from `#components`

File: packages/nuxt-ripple/components/TideDynamicComponents.vue [hunk 3]
```diff
@@ -1,6 +1,9 @@
 <script setup lang="ts">
-import type { TideDynamicPageComponent, TideDynamicComponentGroup } from '../types'
-import { RplPageComponent, RplCardGrid } from '@dpc-sdp/ripple-ui-core'
+import type {
+  TideDynamicPageComponent,
+  TideDynamicComponentGroup
+} from '../types'
+import { RplPageComponent, RplCardGrid } from '@dpc-sdp/ripple-ui-core/vue'
```
Comment (dylankelly) [inline packages/nuxt-ripple/components/TideDynamicComponents.vue:6]: import from `#components`

File: packages/nuxt-ripple/components/TideErrorPage.vue [hunk 4]
```diff
@@ -17,7 +25,7 @@
 </template>
 
 <script setup lang="ts">
-import { RplHeaderGraphic, RplErrorMessage } from '@dpc-sdp/ripple-ui-core'
+import { RplHeaderGraphic, RplErrorMessage } from '@dpc-sdp/ripple-ui-core/vue'
```
Comment (dylankelly) [inline packages/nuxt-ripple/components/TideErrorPage.vue:28]: import from `#components`

File: packages/nuxt-ripple/error.vue [hunk 5]
```diff
@@ -17,7 +25,7 @@
 </template>
 
 <script setup lang="ts">
-import { RplHeaderGraphic, RplErrorMessage } from '@dpc-sdp/ripple-ui-core'
+import { RplHeaderGraphic, RplErrorMessage } from '@dpc-sdp/ripple-ui-core/vue'
```
Comment (dylankelly) [inline packages/nuxt-ripple/error.vue:28]: import from `#components`

**Review by dylankelly — CHANGES_REQUESTED** [review]
@waitingallday can you change those imports in the nuxt only components to import from `#components` instead of `'@dpc-sdp/ripple-ui-core/vue'`

**Review by dylankelly — APPROVED** [review]

**Review by jeffdowdle — APPROVED** [review]

**Pull request merged**